src/icons/SealGrid.tsx

```diff
@@ -8,12 +8,7 @@ const strokeTertiaryLine = stroke('stroke-tertiary')
 
 export default function () {
   return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      width="406"
-      height="126"
-      viewBox="0 0 406 126"
-    >
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 406 126">
```

Comment:
In this file you have `stroke-linecap="round"`, but this property is set by default from `index.css`, remove them please